kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/MovieStore.java

```diff
@@ -0,0 +1,43 @@
+package com.kodilla.good.patterns.challenges;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class MovieStore {
+
+    public Map<String, List<String>> getMovies() {
+
+        List<String> ironManTranslations = new ArrayList<>();
+        ironManTranslations.add("Żelazny Człowiek");
+        ironManTranslations.add("Iron Man");
+
+        List<String> avengersTranslations = new ArrayList<>();
+        avengersTranslations.add("Mściciele");
+        avengersTranslations.add("Avengers");
+
+        List<String> flashTranslations = new ArrayList<>();
+        flashTranslations.add("Błyskawica");
+        flashTranslations.add("Flash");
+
+        Map<String, List<String>> booksTitlesWithTranslations = new HashMap<>();
+        booksTitlesWithTranslations.put("IM", ironManTranslations);
+        booksTitlesWithTranslations.put("AV", avengersTranslations);
+        booksTitlesWithTranslations.put("FL", flashTranslations);
+
+        return booksTitlesWithTranslations;
+    }
+
+    public static void main(String[] args) {
+
+        MovieStore store = new MovieStore();
+
+        String titles = store.getMovies().entrySet().stream()
```

Comment:
Ponieważ klucze w mapie nie mają w tym zadaniu znaczenia, możesz to samo osiągnąć nieco prościej przy pomocy `values()` zamiast `entrySet()`.